src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpIntelliSense.cs

```diff
@@ -99,7 +101,7 @@ public void CtrlAltSpace()
             VisualStudio.Editor.Verify.CurrentLineText("System.Console.WriteLine();$$", assertCaretPosition: true);
             VisualStudio.Editor.SendKeys(VirtualKey.Home, Shift(VirtualKey.End), VirtualKey.Delete);
 
-            VisualStudio.ExecuteCommand(WellKnownCommandNames.Edit_ToggleCompletionMode);
+            VisualStudio.Editor.SendKeys(new KeyPress(VirtualKey.Space, ShiftState.Ctrl | ShiftState.Alt));
```

Comment:
Change is fine but just checking here -- did we break the command?

Reply:
No, the command works, and we use the command in other contexts. This was supposed to be a test for Ctrl+Alt+Space though.